Gemfile

```diff
@@ -13,6 +13,9 @@ gem 'bootsnap', '>= 1.1.0', require: false
 gem 'haml-rails'
 
 group :development, :test do
+  gem 'cucumber-rails',require: false
+  gem 'database_cleaner'
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```

Comment:
Why byebug?

Reply:
Sorry I deleted that but it came back when I merged it, but I have removed it again